speech/grpc/src/test/java/com/examples/cloud/speech/StreamingRecognizeClientTest.java

```diff
@@ -52,11 +52,13 @@
  */
 @RunWith(JUnit4.class)
 public class StreamingRecognizeClientTest {
-  private TestAppender appender;
+  private Writer writer;
+  private WriterAppender appender;
 
   @Before
```

Comment:
I have to do it for each test so that StringWriter is new. There is an After which I now think should be AfterClass where all added appenders will be removed.